src/assets/i18n/en.json

```diff
@@ -160,6 +160,8 @@
       "contributorsAmount": "How many users contributed more than once?",
       "returningContributors": "Returning Contributors",
       "noRanking": "Before April 2018 contributors ranking is not available",
+      "score": "Score",
+      "viewDetails": "View Details",
```

Comment:
@diamond-darrell - should be "Contributor Details"

Reply:
fixed